Modules/System/SharePoint/src/helpers/SharePointRequestHelper.Codeunit.al

```diff
@@ -70,12 +84,11 @@ codeunit 9109 "SharePoint Request Helper"
             if SharePointHttpContent.GetContentType() <> '' then
                 Headers.Add('Content-Type', SharePointHttpContent.GetContentType());
 
-            Headers.Add('X-RequestDigest', SharePointHttpContent.GetRequestDigest());
             RequestMessage.Content(HttpContent);
         end;
     end;
 
-    [NonDebuggable]
+    //[NonDebuggable]
     local procedure SendRequest(HttpRequestMessage: HttpRequestMessage) OperationResponse: Codeunit "SharePoint Operation Response"
```

Comment:
I guess this change was only made for debugging purpose and should ber reverted?